include/hashinator/hashinator.h

```diff
@@ -453,7 +442,7 @@ class Hashmap {
          if (prefetches) {
             buckets.optimizeGPU(s);
          }
-         DeviceHasher::reset_all(buckets.data(), buckets.size(), s);
+         DeviceHasher::reset_all(buckets.data(),_mapInfo, buckets.size(), s);
          _mapInfo->fill = 0;
```

Comment:
So here still the line 446 needs to be deleted. I would also suggest making line 447 `set_status` only check fill on host if debug_hashinator is defined, so that we get rid of that page fault as well.

Reply:
OK done.